jekyll/_cci2/building-docker-images.md

```diff
@@ -132,6 +132,7 @@ To specify the Docker version, you can set it as a `version` attribute:
 
 CircleCI supports multiple versions of Docker. The following are the available versions:
 
+- `20.10.18` 
 - `20.10.17` (default)
```

Comment:
```suggestion
- `20.10.18` (default)
- `20.10.17`
```

v20.10.18 is now the default, i believe :)
https://app.circleci.com/pipelines/github/kelvintaywl-cci/docker-clone3-exploration/1/workflows/2c28b614-600e-42d6-ad6b-48d5cc1a70a3/jobs/1?invite=true#step-101-15